<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="4">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>255</width>
             <height>302</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Draw</string>
           </attribute>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>10</y>
              <width>38</width>
              <height>12</height>
             </rect>
            </property>
            <property name="text">
             <string>Brush</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>70</y>
              <width>38</width>
              <height>12</height>
             </rect>
            </property>
            <property name="text">
             <string>Figure</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>140</y>
              <width>38</width>
              <height>12</height>
             </rect>
            </property>
            <property name="text">
             <string>Measure</string>
            </property>
           </widget>
           <widget class="QToolButton" name="ColortoolButton">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>220</y>
              <width>131</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Color</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/color.png</normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/color.png</iconset>
            </property>
           </widget>
           <widget class="QToolButton" name="BrushtoolButton">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>30</y>
              <width>91</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>Brush</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/draw.png</normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/draw.png</iconset>
            </property>
           </widget>
           <widget class="QSpinBox" name="spinBox">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>30</y>
              <width>51</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QToolButton" name="ColortoolButton_5">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>96</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/arrow2.png</normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/arrow2.png</iconset>
            </property>
           </widget>
           <widget class="QToolButton" name="ColortoolButton_3">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>96</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/triangle2.png</normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/triangle2.png</iconset>
            </property>
           </widget>
           <widget class="QToolButton" name="ColortoolButton_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>96</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/circle2.png</normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/circle2.png</iconset>
            </property>
           </widget>
           <widget class="QToolButton" name="ColortoolButton_4">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>96</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/rectangle2.png</normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/rectangle2.png</iconset>
            </property>
           </widget>
           <widget class="QToolButton" name="ColortoolButton_10">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>160</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/group-14-copy-2@2x.png</normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/group-14-copy-2@2x.png</iconset>
            </property>
           </widget>
           <widget class="QToolButton" name="ColortoolButton_11">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>160</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/group-15-copy@2x.png</normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/group-15-copy@2x.png</iconset>
            </property>
           </widget>
           <widget class="QToolButton" name="ColortoolButton_12">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>160</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/triangle2.png</normaloff>../build-DentistApp-Desktop_Qt_6_4_1_MSVC2019_64bit-Debug/triangle2.png</iconset>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>173</width>
             <height>225</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Image Processing</string>
           </attribute>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="patientTableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QMdiArea" name="mdiArea"/>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="toolButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="toolButton_2">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QTreeWidget" name="treeWidget_2">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Button</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Button</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="separator"/>
   <addaction name="actionRedo"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>저장</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
